js/pages/iptt_report/components/report/reportBody.js

```diff
@@ -4,20 +4,29 @@ import ReportTableHeader from './tableHeader';
 import ReportTableBody from './tableBody';
 
 
-export default ({ setBottomScrolling, setExtend }) => {
+export default ({ setBottomScrolling, setExtend, setTableHeight }) => {
 
     useEffect(() => {
-        /* Adding a listener to track the position on page scrolling. Scrollable is the visible area a user can scroll the page calculated by the total height of page minus height of the visible window minus height of the footer. */
-        window.addEventListener('scroll', () => {
+        /* Adding a function to track the position on page scrolling. Scrollable is the visible area a user can scroll the page calculated by the total height of page minus height of the visible window minus height of the footer. */
+        const scrollEventHandler = () => {
             let footer = document.querySelector("#footer").offsetHeight;
             let scrollable = document.documentElement.scrollHeight - window.innerHeight - footer;
+            
+            // Setting the total height of the table. Set time out needed to render the sidebar in correct order when changing programs, otherwise the sidebar's scroll Top will be off.
+            setTimeout(() => {
+                setTableHeight(scrollable);
+            }, 1)
 
             // Setting how much space left a user can scroll to reach the bottom.
-            setBottomScrolling(scrollable - window.scrollY) 
+            setBottomScrolling(parseInt(scrollable - window.scrollY));
```

Comment:
I'm confused about the user of parseInt here. If one of the inputs is text, this parseInt doesn't help, does it?

Reply:
You can use parseInt to drop the decimals. I just use it as an alternative to rounding.

Reply:
Ah, gotcha, that makes sense.